crates/spirv-std/shared/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "spirv-types"
+version = "0.4.0-alpha.3"
+authors = ["Erin Power <erin.power@embark-studios.com>"]
```

Comment:
Nit: `Embark <opensource@embark-studios.com>`. Also `version = "0.4.0-alpha.5"`, and `repository = "https://github.com/EmbarkStudios/rust-gpu"`